### PR TITLE
TASK: Add startingPoint property to menu node type

### PR DESCRIPTION
**Pull request by khuppenbauer**

### CODE REVIEW

File: TYPO3.Neos.NodeTypes/Configuration/NodeTypes.Content.yaml [hunk 1]
```diff
@@ -116,6 +116,13 @@
         label: i18n
         inspector:
           group: 'options'
+    startingPoint:
+      type: 'reference'
+      ui:
+        reloadIfChanged: true
+        label: i18n
+        inspector:
```
Comment (johannessteu) [inline TYPO3.Neos.NodeTypes/Configuration/NodeTypes.Content.yaml]: Could be restricted with `editorOptions` to documents only?

Comment (khuppenbauer) [inline TYPO3.Neos.NodeTypes/Configuration/NodeTypes.Content.yaml]: you're right - just tested it and it seems to be the default. but i can add it

**Comment by kitsunet** [comment]
Good idea, thanks!

**Comment by johannessteu** [comment]
👍  by reading

**Comment by daniellienert** [comment]
👍🏼

**Comment by kdambekalns** [comment]
I wonder if we could put this into the 2.0 branch even… 👼

**Comment by aertmann** [comment]
funny thought this was already there.. so thanks for adding it :)

I'd personally leave out the redundant default node type filter part, but wouldn't make it a blocker..

@kdambekalns: we could, but we shouldn't.. it is easily doable for anyone who needs it in a current project

**Comment by khuppenbauer** [comment]
i can remove the node type filter tonight
by the way - do we need more translations? just added the default label

**Comment by aertmann** [comment]
@khuppenbauer: Thanks. No more translations should be needed.

**Pull request merged**